app/src/main/java/dev/sebastiano/bundel/notificationslist/NotificationItem.kt

```diff
@@ -102,8 +102,8 @@ private fun previewNotification(context: Context) = ActiveNotification(
         )
     ),
     icons = ActiveNotification.Icons(
-        appIcon = Icon.createWithResource(context, R.drawable.ic_default_icon),
-        large = Icon.createWithResource(context, R.drawable.outline_interests_black_48dp)
+        appIcon = Icon.createWithResource(context, dev.sebastiano.bundel.ui.R.drawable.ic_bundel_icon),
```

Comment:
Import?

Reply:
done. Note that this is non-transitive R files, thats why the R is now imported and not shown on the line, i.e. its an R file from another module. It can get confusing when you use R files from multiple modules, people use alias or leave it as it was before with the full import.

<img width="1610" alt="image" src="https://github.com/code-with-the-italians/bundel/assets/655860/8b632e32-2953-43be-a498-d30d7b7e9cd5">

Reply:
I'd be ok with aliases if there were clashes 👍